tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -579,6 +586,18 @@ def _operation_status_done(operation):
     def _execute(  # pylint: disable=arguments-differ
         self, request, *, timeout_sec=_WAIT_FOR_OPERATION_SEC
     ):
+        if self.gfe_debug_header != DISABLE_DEBUG_HEADER_VALUE:
+
+            def _log_debug_header(resp):
```

Comment:
Since we're not using local vars to restore the previous postproc anymore - no need to make it an inner method. Let's make it a class method.

Reply:
done.